src/nvim/screen.c

```diff
@@ -1495,8 +1495,9 @@ static void win_update(win_T *wp)
 
     // make sure the rest of the screen is blank
     // write the 'eob' character to rows that aren't part of the file.
-    win_draw_end(wp, wp->w_p_fcs_chars.eob, ' ', false, row, wp->w_grid.Rows,
-                 HLF_EOB);
+    bool minimal = wp->w_float_config.minimal;
+    win_draw_end(wp, minimal ? ' ' : wp->w_p_fcs_chars.eob, ' ', false, row, wp->w_grid.Rows,
+                 minimal ? 0 : HLF_EOB);
```

Comment:
'fcs' is window-local, so maybe the special case isn't needed here?

Reply:
All options this concerns are window-local, otherwise it would no point of including them. I did it this way for `fcs` because I didn't like going in and string-edit a specific flag in a larger user option.

Reply:
`set fcs-=...` is perfectly reasonable IMO. The main purpose of 'minimal' is to save users from needing to write the equivalent VimL/Lua themselves. So it can be implemented in VimL/Lua (in spirit, or literally).

Reply:
Lets do that then. Though `set winhl+=EndOfBuffer:X` will be a bit more messy, as there is no canonical X to mean "no highlight" (`Normal` doesn't do this in nvim, because reasons). But I'm thinking of allowing `EndOfBuffer:` syntax to do this.